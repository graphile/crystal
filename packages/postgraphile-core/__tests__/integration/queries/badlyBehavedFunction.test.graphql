{
  badlyBehavedFunction {
    nodes {
      ...PersonFragment
    }
    edges {
      cursor
      node {
        ...PersonFragment
      }
    }
  }
}

fragment PersonFragment on Person {
  nodeId
  id
  firstName
}
