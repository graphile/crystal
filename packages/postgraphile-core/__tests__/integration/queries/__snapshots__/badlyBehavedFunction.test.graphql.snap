// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "badlyBehavedFunction": Object {
      "edges": Array [
        Object {
          "cursor": "WyJuYXR1cmFsIiwxXQ==",
          "node": Object {
            "firstName": "John",
            "id": 1,
            "nodeId": "WyJwZW9wbGUiLDFd",
          },
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwyXQ==",
          "node": null,
        },
        Object {
          "cursor": "WyJuYXR1cmFsIiwzXQ==",
          "node": Object {
            "firstName": "Twenty",
            "id": 6,
            "nodeId": "WyJwZW9wbGUiLDZd",
          },
        },
      ],
      "nodes": Array [
        Object {
          "firstName": "John",
          "id": 1,
          "nodeId": "WyJwZW9wbGUiLDFd",
        },
        null,
        Object {
          "firstName": "Twenty",
          "id": 6,
          "nodeId": "WyJwZW9wbGUiLDZd",
        },
      ],
    },
  },
}
`;
